### PR TITLE
cloud-provider-kubevirt: Bootstrapping Configuration for the new Repository

### PR DESCRIPTION
**Pull request by nirarg**
This PR has dependent  on PR in `openshift/cloud-provider-kubevirt`, which adds the dockerfile
https://github.com/openshift/cloud-provider-kubevirt/pull/1/files

### CODE REVIEW

**Review by droslean — APPROVED** [review]

File: core-services/prow/02_config/openshift/cloud-provider-kubevirt/_prowconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,114 @@
+tide:
+  queries:
+  - includedBranches:
```
Comment (dhellmann) [inline core-services/prow/02_config/openshift/cloud-provider-kubevirt/_prowconfig.yaml:3]: Do all of these branches actually exist? If they don't exist, do we need them in this list?

File: core-services/prow/02_config/openshift/cloud-provider-kubevirt/_prowconfig.yaml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - keep-main-query-separate
+    - needs-rebase
+    repos:
+    - openshift/cloud-provider-kubevirt
+  - excludedBranches:
```
Comment (dhellmann) [inline core-services/prow/02_config/openshift/cloud-provider-kubevirt/_prowconfig.yaml:74]: Same question about whether these branches apply.

**Review by dhellmann — COMMENTED** [review]
I'm not an expert in this repo, so may not be the best reviewer for this PR. I have a couple of inline questions, but before I approve I'd like to see some reviews from other folks who know more about the details of Prow config.

File: core-services/prow/02_config/openshift/cloud-provider-kubevirt/_prowconfig.yaml (hunk 1 above)
Comment (nirarg) [reply core-services/prow/02_config/openshift/cloud-provider-kubevirt/_prowconfig.yaml:3]: Thank you @dhellmann 
This was actually auto generated and I'm not sure if I need or can change it
@droslean can you please advise on this?

**Comment by nirarg** [comment]
> /approve

Thank you @droslean 
@dhellmann can you please approve this?

**Pull request merged**